src/components/FullTags/TagsCombinaisonsForm.vue

```diff
@@ -0,0 +1,378 @@
+                    formated_tags[tag.tag.toLowerCase()] = {
+                        nb_images: tag.totalImage
+                    }
+                })
+                this.sortInTwoArrays(formated_tags, this.selected_tags[0].toLowerCase())
+                return this.images
+            }
+            const tags_without_input = {}
+            //Get only images including all selected tags except first (user current input) in array of selected tags
+            const images_with_tags = this.images.filter((im) => {
+                let ok = true
+                let i = 1;
+                //Check all selected tags except first (user current input) if in image tags
+                do {
+                    //Management of case sensitivity for UX
+                    const img_lower_case_tags = []
+                    im.tags.forEach(tg => {
+                        img_lower_case_tags.push(tg.toLowerCase())
+                    })
+                    // ensure current tag is in image tags
+                    if (img_lower_case_tags.includes(this.selected_tags[i].toLowerCase())) {
+                        img_lower_case_tags.forEach(img_tag => {
+                            //Ensure all  confirmed selected tags are into the image tags before adding this image to the concerned tags possibilities.
+                            if (i == this.selected_tags.length - 1) {
+                                // Case 1, this tag was already in a image to keep
+                                if (tags_without_input[img_tag]) {
+                                    tags_without_input[img_tag].nb_images += 1
+                                }
+                                // Case 2, no image parcoured had the tag to add yet
+                                else if (!tags_without_input[img_tag]) {
+                                    tags_without_input[img_tag] = { nb_images: 1 }
+                                }
+                            }
+                        })
+                    } else {
+                        ok = false
+                    }
+                    i++;
+                } while (ok && i < this.selected_tags.length)
+                this.selected_tags_with_images = tags_without_input
+                return ok
+            })
+            this.sortInTwoArrays(this.selected_tags_with_images, this.selected_tags[0].toLowerCase())
+            // console.log("LES IMAGES:", this.selected_tags_with_images, "Les images + input:", this.input_and_selected_tags_with_images, "input: ", this.selected_tags[0])
+            return images_with_tags
+        },
+        confirmed_tags() {
+            const formated_selected_tags = []
+            for (let i = 1; i < this.selected_tags.length; i++) {
+                formated_selected_tags.push(this.selected_tags[i])
+            }
+            return formated_selected_tags
+        },
+        ...mapGetters({
+            selected_tags: "getTags",
+            im_2: "getImages"
```

Comment:
im_2? what is im_1? What is the purpose of im_2. Please give a good name to your function and variable